ntbs-service/Models/NtbsContext.cs

```diff
@@ -19,6 +19,7 @@ public NtbsContext(DbContextOptions<NtbsContext> options)
 
         public virtual DbSet<Country> Country { get; set; }
         public virtual DbSet<Ethnicity> Ethnicity { get; set; }
+        public virtual DbSet<ClinicalTimeline> ClinicalTimelines { get; set; }
```

Comment:
This is actually superfluous as you can't create db sets for owned types, see my comment above - it's not used anywhere I added it before I made it an owned type!